packages/utils/src/Handlers.ts

```diff
@@ -1,10 +1,23 @@
 import { wrapHookToRecognizeElement, Connector } from "./wrapConnectorHooks";
 
+// TODO(mat) any reason why why use array to store this information ?
```

Comment:
@prevwong not sure if this structure is easy to read, any arguments why not use object instead of array?

Reply:
I just felt that in the class that would extend the Handlers class, typing it in an object form would be a bit less intuitive:

```
select: {
 events: [
    {
       name: "mousedown",
       handler: () => {...},
       capture: true
    }
 ]
}
```

Additionally, I wanted to have it resemble the native `addEventListener` API which an array was better able to achieve that `['mousedown', () => {...}, true]`

Reply:
I see, well that makes sense, but now we lost this sync, since the third argument became an object so its no longer possible to use them as-is, maybe worth re-evaluating?

Reply:
Yeah I agree, the third argument becoming an object takes away the magic, and we have the `defineEventListener` util which makes things easier for developers. 

But perhaps let's keep that in a separate PR? We need to update the `LayerHandlers` to use the `defineEventListener` util as well  (it's currently using the array form right now)